examples/compute/instance.gyro

```diff
@@ -0,0 +1,76 @@
+zone: "us-west1-a"
+
+google::compute-disk instance-disk-example
+    name: "instance-disk-example"
+    description: "Gyro Instance Test Disk"
+    zone: $zone
+    size-gb: 32
+    type: "pd-standard"
+    physical-block-size-bytes: 4096
+end
+
+google::instance gyro-dev-1
+    name: "gyro-development"
+    description: "Testing for Gyro"
+    zone: $zone
+    machine-type: "zones/$(zone)/machineTypes/n1-standard-1"
+
+    network-interfaces
+        network: $(external-query google::compute-network {name: "default"})
+    end
+
+    initialize-disks
+        boot: true
+
+        initialize-params
+            disk-name: "gyro-boot-disk"
+            source-image: "projects/debian-cloud/global/images/family/debian-9"
+        end
+    end
+
+    initialize-disks
+        initialize-params
+            disk-name: "gyro-secondary-disk"
+            source-image: "projects/debian-cloud/global/images/family/debian-9"
+        end
+    end
+
+    labels: {
+        "gyro": "install"
+    }
+end
+
+google::instance gyro-dev-2
+    name: "gyro-development-2"
+    description: "Testing for Gyro"
+    zone: $zone
+    machine-type: "zones/$(zone)/machineTypes/n1-standard-1"
+
+    network-interfaces
+        network: $(external-query google::compute-network {name: "default"})
+    end
+
+    initialize-disks
+        boot: true
+
+        initialize-params
+            disk-name: "gyro-boot-disk-2"
+            source-image: "projects/debian-cloud/global/images/family/debian-9"
+        end
+    end
+
+    labels: {
+        "gyro": "install"
+    }
+end
+
+google::instance-attached-disk gyro-disk-1
+    instance: $(google::instance gyro-dev-1)
+    attached-disk
+        auto-delete: false
+        source: $(google::compute-disk instance-disk-example)
+    end
+end
+
+instance: $(external-query google::instance {zone: $zone, filter: 'name = gyro-development'})
+instances: $(external-query google::instance)
```

Comment:
external queries should not be part of the example config file

Reply:
Why not show the example usage of the finder implementation?

Reply:
Just for consistency as we don't have it anywhere else. Also one example is part of the finder doc

Reply:
Done.